### PR TITLE
initial branch commit edit in README.md

### PR DESCRIPTION
**Pull request by j-devy**

### CODE REVIEW

File: client/js/array-util.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+      .map(charCode => String.fromCharCode(charCode));
 }
 
 const sumRow = function(array) {
-	var sum = '' || 0;
-	for (var i = 0; i < array.length; i++) {
-		sum += array[i];
-	}
-	return sum;
+  var sum = '' || 0;
```
Comment (speschellk) [inline client/js/array-util.js]: Beware the "var"! With ES6's "const" and "let" identifiers, you can now replace ever "var" with one of those preferable options. Please do that throughout.

File: client/js/table-view.js [hunk 2]
```diff
@@ -2,122 +2,121 @@ const { getLetterRange, sumRow } = require('./array-util');
...
+    for (let row = 0;  row < this.model.numRows; row++) {
+      const tr = createTR();
+      for (let col = 0; col < this.model.numCols; col++) {
+        const position = {col: col, row: row};
+        const value = this.model.getValue(position);
+        const td = createTD(value);
+
+
+        if (typeof value !== 'undefined' && curCol === col && !isNaN(value)) {
+            sum = sum + parseInt(value);
```
Comment (speschellk) [inline client/js/table-view.js]: A way to abbreviate this is "sum += parseInt(value);"

File: client/js/test/fixtures/sheet-container.html [hunk 3]
```diff
@@ -1,17 +1,17 @@
 <div class="container">
-	<input id="formula-bar" type="text" value="" />
-	<table id="sheet-current">
-		<thead>
-			<tr></tr>
-		</thead>
+  <input id="formula-bar" type="text" value="" />
+  <table id="sheet-current">
+    <thead>
+        <tr></tr>
```
Comment (speschellk) [inline client/js/test/fixtures/sheet-container.html]: Indentation issue

File: client/js/test/fixtures/sheet-container.html [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+    </thead>
 
-		<tfoot>
-			<tr></tr>
-		</tfoot>
+    <tfoot>
+        <tr></tr>
```
Comment (speschellk) [inline client/js/test/fixtures/sheet-container.html]: Indentation issue (too far)

File: client/js/test/fixtures/sheet-container.html [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+    </tfoot>
 
-		<tbody>
-			<tr></tr>
-		</tbody>
-		
-	</table>
+    <tbody>
+        <tr></tr>
```
Comment (speschellk) [inline client/js/test/fixtures/sheet-container.html]: Indentation issue

File: client/js/table-view.js (hunk 2 above)
Comment (speschellk) [inline client/js/table-view.js]: Did you see this note?

File: client/js/table-model.js [hunk 6]
```diff
@@ -1,21 +1,21 @@
 class TableModel {
-	constructor(numCols=10, numRows=20) {
-		this.numCols = numCols;
-		this.numRows = numRows;
-		this.data    = {};
-	}
+  constructor(numCols=10, numRows=20) {
```
Comment (speschellk) [inline client/js/table-model.js]: Please use one space around all operators to improve readability.

File: client/js/array-util.js (hunk 1 above)
Comment (speschellk) [inline client/js/array-util.js]: Did you see this note?

Comment (j-devy) [inline client/js/array-util.js]: yes i this note. i'll make the changes right now

File: client/js/table-view.js (hunk 2 above)
Comment (j-devy) [inline client/js/table-view.js]: yep i see this note

**Pull request merged**